testdata/networking/multus-cni/Pods/IPv6/1interface-macvlan-bridge-v6.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  generateName: macvlan-bridge-pod-v6
+  labels:
+    name: macvlan-bridge-pod-v6
+  annotations:
+    k8s.v1.cni.cncf.io/networks: macvlan-bridge-v6
+spec:
+  containers:
+  - name: macvlan-bridge-pod-v6
+    image: docker.io/aosqe/multus-pod:latest
```

Comment:
can we replace the image with 'aosqe/pod-for-ping' and with digest.
eg: docker.io/aosqe/pod-for-ping@sha256:f61ca8fd1dd7dfb8d403b66df549ed5a94fc4626d3b19e2530af48aefa75e5ea
since this images has been mirror for disconnect cluster